### PR TITLE
Mode Switching and subscribing

### PR DESCRIPTION
**Pull request by japneetkaurbhatia**

### CODE REVIEW

File: src/controllers/SubscribeController.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ import GenericResponse from '../utilities/GenericResponse';
 import SubscribeService from '../services/SubscribeService';
 
 const subscribe = async (req, res) => {
+  console.log('IN CONTROLLER \n');
```
Comment (hariharanweb) [inline src/controllers/SubscribeController.js]: Remove this

File: src/services/RegistryService.js [hunk 2]
```diff
@@ -1,13 +1,22 @@
...
+const getRegistry = (filter) => {
+  const logger = LoggingService.getLogger('RegistryService');
+  const filename = fileURLToPath(import.meta.url);
+
+  const dirname = path.dirname(filename);
+  const registryFilePath = path.join(dirname, '../registry/registry.json');
+
+  const value = fs.readFileSync(registryFilePath, { encoding: 'utf8' });
+  const registry = JSON.parse(value);
+  logger.debug(`In get registry: \n${JSON.stringify(registry)}`);
```
Comment (hariharanweb) [inline src/services/RegistryService.js]: Remove this

**Review by hariharanweb — CHANGES_REQUESTED** [review]

File: src/controllers/SubscribeController.js (hunk 1 above)
Comment (japneetkaurbhatia) [reply src/controllers/SubscribeController.js]: Updated

File: src/services/RegistryService.js (hunk 2 above)
Comment (japneetkaurbhatia) [reply src/services/RegistryService.js]: Updated

**Pull request merged**